onnxruntime/core/optimizer/qdq_transformer/selectors_actions/shared/utils.h

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <string>
+#include "core/graph/basic_types.h"
+#include "core/providers/nnapi/nnapi_builtin/nnapi_lib/NeuralNetworksTypes.h"
+#include "../qdq_selectors.h"
+#include "../../../selectors_actions/helpers.h"
+
+namespace onnxruntime {
+
+class GraphViewer;
+class Node;
+
+namespace QDQ {
+
+struct Selector {
+  using OpVersionsMap = std::unordered_map<std::string, std::vector<ONNX_NAMESPACE::OperatorSetVersion>>;
```

Comment:
It's a little confusing to have a struct called Selector that is constructed with a BaseSelector.

It's providing a join between the selector and the op+versions supported, so the name should probably suggest that more. e.g. something like OpVersionsAndSelector